### PR TITLE
Fixing github actions and windows build script

### PR DESCRIPTION
**Pull request by andersonpablosilva**
Updating github actions and windows build script after github security update.

### CODE REVIEW

File: scripts/build/compile/build.bat [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 pipenv uninstall --all
 pipenv install --ignore-pipfile --dev
+pipenv install pywin32-ctypes
+pipenv install pyinstaller
```
Comment (diego-pedroso) [inline scripts/build/compile/build.bat:9]: I believe this one is duplicated.

https://github.com/HPInc/ml-git/blob/e343d634d51f4375e816adb3e4554046103c1b43/Pipfile#L11

File: scripts/build/compile/build.bat [hunk 2]
```diff
@@ -5,6 +5,8 @@
 
 pipenv uninstall --all
 pipenv install --ignore-pipfile --dev
+pipenv install pywin32-ctypes
```
Comment (diego-pedroso) [inline scripts/build/compile/build.bat:8]: This one should be version locked and moved to Pipfile

File: scripts/build/compile/build.bat (hunk 1 above)
Comment (lucasncabral) [reply scripts/build/compile/build.bat]: The command was deleted in 40fddf3.

File: scripts/build/compile/build.bat (hunk 2 above)
Comment (lucasncabral) [reply scripts/build/compile/build.bat]: Done in 40fddf3.

File: scripts/build/compile/build.bat [hunk 3]
```diff
@@ -4,7 +4,7 @@
 ::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
 
 pipenv uninstall --all
-pipenv install --ignore-pipfile --dev
+pipenv install --dev
```
Comment (diego-pedroso) [inline scripts/build/compile/build.bat:7]: Please revert this change.

**Review by diego-pedroso — APPROVED** [review]
